prow/plugins.yaml

```diff
@@ -16,6 +16,7 @@ triggers:
 - repos:
   - google/cadvisor
   - GoogleCloudPlatform/k8s-multicluster-ingress
+  - GoogleCloudPlatform/netd
```

Comment:
re: I guess trusted_org is for the k8s member, and it won't need the user to be k8s members, right?

If you don't want this restriction, just drop the line.

Reply:
oh, I noticed trusted_org for the previous repos section, didn't see this one also have it.
Thanks